adapters/dynamoDB_adapters/src/main/java/org/atomhopper/dynamodb/query/CategoryStringGenerator.java

```diff
@@ -0,0 +1,160 @@
+package org.atomhopper.dynamodb.query;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public final class CategoryStringGenerator {
+
+    private static final char INCLUSIVE_OPERATOR = '+', ESCAPE_OPERATOR = '\\';
+    private static final char[] OPERATORS = {INCLUSIVE_OPERATOR, ESCAPE_OPERATOR};
+
+    private CategoryStringGenerator() {
+        throw new AssertionError();
+    }
+
+    /**
+     * Takes a search string, prefix map & prefix character & returns an array of strings which are arguments to the
+     * corresponding generated SQL statement.
+     *
+     * See SearchToSqlConverter for more details.
+     *
+     * @param searchString
+     * @param mapPrefix
+     * @param prefixSplit
+     *
+     * @return
+     */
+    public static List<String> getPostgresCategoryString(String searchString, Map<String, String> mapPrefix, String prefixSplit ) {
```

Comment:
It would be good if the method can be simplified to increase readability.

Reply:
ok.will do the changes